test/services/edusharing/index.test.js

```diff
@@ -0,0 +1,12 @@
+const { expect } = require('chai');
+const app = require('../../../src/app');
+
+// const { someHelperFunction } = require('./helper');
+
+const eduSearch = app.service('edusharing/eduSearch');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['eduSearch' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)